packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/MakeOffer/index.tsx

```diff
@@ -53,7 +53,12 @@ const MakeOffer: React.FC<Props> = (props) => {
     walletCurrency
   } = props
   const [termsAccepted, setTermsAccepted] = useState(false)
-
+  useEffect(() => {
+    props.analyticsActions.trackEvent({
```

Comment:
You sure you want this event to fire every time this component re-renders? You might want to double check to see if it's being re-rendered multiple times due to things like the form changes, etc.

Reply:
Just want it to render on mount (whenever they view this page).  I think the empty dependency array makes it just the one render?

Reply:
Yup